src/main/resources/assets/opencomputers/doc/en_us/item/componentBusCreative.md

```diff
@@ -0,0 +1 @@
+#REDIRECT componentBus1.md
```

Comment:
The actual content of the file is fine, but the path of the file should be in the en_US folder and not en_us (capitalization). It also looks like 1.7.10 had the same problem.

Reply:
Oh, the joy of backporting on Windows. I probably forgot to reset my index before committing, so the 1.12 capitalization stayed. I'm pretty sure this'll cause issues on Linux, so it should be fixed.